agent/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "license-manager-agent"
-version = "3.0.9"
+version = "3.0.10-alpha.1"
```

Comment:
Should this be `3.0.10a1`?

Reply:
there's no standard across our services actually. Some use this nomenclature, other use `ax`. Anyway for me, I will change